packages/react-experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -145,7 +147,9 @@ const SelectedPersonaInner = React.memo(
       [selected, isValid, theme],
     );
 
-    const coinProps = {};
+    const defaultPersonaSize = PersonaSize.size32;
+    const personaSize = size !== undefined ? size : defaultPersonaSize;
```

Comment:
nit: this syntax will also work and is shorter/seems to be used more often:

const personaSize = size | PersonaSize.size32

Reply:
OH! That's neat! I like it!

But wait, doesn't that reject all falsey values? As in, it will reject the `0` valued enum value? Hmm, in this case `tiny` is deprecated, so we don't have to worry about it.

I'll change it.

Reply:
This no longer applies, since this code is no longer necessary. Resolving.